outputs/core_pools.json

```diff
@@ -67,12 +67,11 @@
   },
   "zkevm": {
     "0x1d0a8a31cdb04efac3153237526fb15cc65a252000000000000000000000000f": "B-rETH-STABLE",
-    "0x65da876a95cd5b6a5880710628c436409b1b29fa00000000000000000000005b": "wstETH/ankrETH ",
-    "0xb1557cfea06de5a1601a7f0ccf3b515ef431a50d000200000000000000000059": "ECLP-wETH-rETH",
-    "0xdf725fde6e89981fb30d9bf999841ac2c160b512000000000000000000000010": "B-wstETH/rETH-STABLE",
```

Comment:
Here are some examples of pools that are right on the line.  I'm not sure we should be removing core pools that were over 250k and fall under/we need to think about this edge case.

Reply:
only the last one is arguably on the line, first has a tvl of 11.7k and should obviously get removed. second one tvl of 121k, idem. last one is at 240k. needs a slight bump to get back into core pools

i think this problem will always arise when drawing a line somewhere, there will be pools right under or over